### PR TITLE
Release 1.1.1

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by ypriverol** [comment]
@fabianegli suggested doing a release 1.1.1 including the  following fixs: 
- changes in Comet version to 2023
- bug fixing from @sminot about container resolve in AWS. (https://github.com/nf-core/quantms/pull/93)
- Zenodo accession in README. 
- MicroMamba for CI/CD.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] nfcore/quantms - [TBD] - Thimphu
```
Comment (jpfeuffer) [inline CHANGELOG.md:6]: Shouldn't this be 1.1.1?
I think it doesn't hurt adding a small changelog for it.

- Updated Comet version.
- Improved output docs for mzTab
- Added dev profile for nightly versions of OpenMS tools
- Fixed bug in docker vs. singularity container logic in some processes.

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

**Comment by jpfeuffer** [comment]
Yes please combine with my suggestion s

File: CHANGELOG.md (hunk 1 above)
Comment (ypriverol) [reply CHANGELOG.md:6]: @jpfeuffer this is a minor update, and bug fix, then probably 1.1.1 looks more appropriate no? My point is, the previous 1.1.0 is really a major compare with 1.0.0.

Comment (ypriverol) [reply CHANGELOG.md:6]: Sorry, I misunderstood your comment. You are right.

File: nextflow.config [hunk 2]
```diff
@@ -366,8 +376,8 @@ manifest {
     description     = """Quantitative Mass Spectrometry nf-core workflow"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=22.10.1'
-    version         = '1.1.0'
-    doi             = ''
+    version         = '1.1.1dev'
```
Comment (fabianegli) [inline nextflow.config:379]: We're getting ready for the release, so it makes sense now to set the release version.

**Review by fabianegli — CHANGES_REQUESTED** [review]
Apart form the small changes, it is ready for release.

**Review by jpfeuffer — APPROVED** [review]

**Review by fabianegli — APPROVED** [review]

**Pull request merged**